examples/README.md

```diff
@@ -4,3 +4,14 @@
 - [uprobe](uprobe/) - Like a kprobe, but for symbols in userspace binaries (e.g. `bash`).
 - [tracepoint](tracepoint/) - Attach a program to predetermined kernel tracepoints.
 - Add your use case(s) here!
+
+## How to run
+
+```bash
+git clone https://github.com/cilium/ebpf.git
+cd examples/[kprobe,uprobe,tracepoint...]
```

Comment:
You can simplify this to:

```suggestion
cd ebpf/examples/
go run -exec sudo [kprobe,uprobe,tracepoint...]
```

No go build step needed, so you can skip the the bit about `go build main.go`.

Reply:
I tried "go run -exec sudo kprobe" but got errors: `package kprobe is not in GOROOT (/snap/go/7360/src/kprobe)`. And I tried "go run -exec sudo ./kprobe" and it works. So I will document that instead.

Reply:
Thanks!